requirements-dev.txt

```diff
@@ -1,3 +1,6 @@
 requests==2.22.0
 pytest==5.2.1
-twine==2.0.0
\ No newline at end of file
+twine==2.0.0
+
+psycopg2-binary==2.8.4
+fhirbase==0.0.1 
```

Comment:
I think we should add a `requirements-benchmark.txt` to split the dependancies really useful to the project and those only useful for benchmark things

Reply:
:+1: